homeassistant/components/blue_current/strings.json

```
@@ -136,5 +146,28 @@
         "name": "Block charge point"
       }
     }
+  },
+  "selector": {
+    "select_charging_card": {
+      "options": {
+        "without_charging_card": "Without charging card"
+      }
+    }
+  },
+  "services": {
+    "start_charge_session": {
+      "name": "Start charge session",
+      "description": "Start a new charge session.",
```

Small nitpick: We generally use third person singular for action descriptions to ensure correct translations.
Also we should add a few words so it's not just paraphrasing the name by including some info about mandatory fields:

```suggestion
      "description": "Starts a new charge session on a specified charge point.",
```